docs/issues-and-help.txt

```diff
@@ -0,0 +1,56 @@
+.. _laravel-issues-and-help:
+
+=============
+Issues & Help
+=============
+
+We are lucky to have a vibrant PHP community that includes users of varying
+experience with MongoDB PHP Library and {+odm-short+}. To get support for
+general questions, search or post in the
+:community-forum:`MongoDB PHP Community Forums </tag/php>`.
+
+To learn more about MongoDB support options, see the
+`Technical Support <https://www.mongodb.org/about/support>`__ page.
+
+Bugs / Feature Requests
+-----------------------
+
+If you find a bug or want to see a new feature in {+odm-short+},
+please report it as a GitHub issue in the `mongodb/laravel-mongodb
+<https://github.com/mongodb/laravel-mongodb>`__ repository.
```

Comment:
Since the repository URL is referenced a few times on this page (possibly elsewhere), would it make sense to define this somewhere so we can avoid repetition? I'm thinking similar to what exists for `:community-forum:` above, which seems to also support appending to the base URL.

Reply:
^ you can define a source constant for the url stem like `mongodb-laravel-gh` or something!